### PR TITLE
Tempo loader

### PR DESCRIPTION
**Pull request by bmcfee**
implements #183

### CODE REVIEW

File: mir_eval/io.py [hunk 1]
```diff
@@ -463,6 +464,50 @@ def load_key(filename, delimiter=r'\s+'):
...
+    denotes the relative confidence of the first value compared to the
+    second (in the range [0, 1]). The file should contain only one row.
+
+    Parameters
+    ----------
+    filename : str
+        Path to the annotation file
+    delimiter : str
+        Separator regular expression.
+        By default, lines will be split by any amount of whitespace.
```
Comment (craffel) [inline mir_eval/io.py]: Nitpick: "lines" is a bit of a misnomer here, since we require that there only be a single line.

**Comment by craffel** [comment]
Thank you!  Made one tiny docstring nitpick, otherwise LGTM.  Feel free to fix and merge at will.

File: mir_eval/io.py (hunk 1 above)
Comment (bmcfee) [inline mir_eval/io.py]: This should be tokens/columns, right?

Comment (craffel) [inline mir_eval/io.py]: Depends on your perspective I guess. Is it the line that's getting split into tokens/columns, or are the tokens/columns being solit out of the line or something?

Comment (bmcfee) [inline mir_eval/io.py]: Er, I stole that comment from the key loader... sounds like your fault :grin: 

But in general, it's the line(s) being split into columns.  I think the existing description is correct.

**Comment by craffel** [comment]
Merged, thanks again.

**Pull request merged**